integrations/completions/ort-completion.fish

```diff
@@ -25,7 +25,7 @@ complete -c ort -n "__fish_seen_subcommand_from advise" -l output-dir -s o -r -F
 complete -c ort -n "__fish_seen_subcommand_from advise" -l output-formats -s f -r -fa "JSON YAML" -d 'The list of output formats to be used for the ORT result file(s).'
 complete -c ort -n "__fish_seen_subcommand_from advise" -l label -s l -r -d 'Set a label in the ORT result, overwriting any existing label of the same name. Can be used multiple times. For example: --label distribution=external'
 complete -c ort -n "__fish_seen_subcommand_from advise" -l resolutions-file -r -F -d 'A file containing issue and rule violation resolutions.'
-complete -c ort -n "__fish_seen_subcommand_from advise" -l advisors -s a -r -d 'The comma-separated advisors to use, any of [BlackDuck, OSSIndex, OSV, VulnerableCode].'
+complete -c ort -n "__fish_seen_subcommand_from advise" -l advisors -s a -r -d 'The comma-separated advisors to use, any of [BlackDuck, OssIndex, Osv, VulnerableCode].'
```

Comment:
I still find it nicer to use the correct spelling also for IDs, like "OSV" instead of "Osv", even if in the end IDs are handled case-insensitively.
More importantly, I'm a bit concerned that changing the default spelling of plugin IDs could cause issues in the ORT Server database because I don't think IDs are handled case-insensitively there. It could also be fine, I just don't have time to check this now.

Reply:
Yes, I would actually be in favor of not changing any plugin IDs right now. I have added a sentence to https://github.com/eclipse-apoapsis/ort-server/issues/1821 to verify that changing only the spelling of plugin IDs does not cause issues.

Reply:
I'd like to finish this up. Please have another look @mnonnenmacher.